tests/components/ecobee/test_util.py

```diff
@@ -0,0 +1,34 @@
+"""Tests for the ecobee.util module."""
+import unittest
+import voluptuous as vol
```

Comment:
Please add blank lines between standard library, 3rd party and homeassistant import groups.